### DISCUSSION TITLE
Sending Notifications by DBADash

### DISCUSSION THREAD

**Discussion by gayazics** [trimble-oss/dba-dash#157] [Polls]
Hi, 

The tool is collecting good number of details but it does not have ability to send notifications to the groups\team in case of any detail reaching the threshold etc. For anyone who would like to implement this in production environments with good number of instances being monitored, sending notifications is really required.

**Comment by DavidWiseman** [comment]
Hi,

I've just [added some documentation](https://github.com/trimble-oss/dba-dash/blob/main/Docs/Alerts.md) on alerts.  DBA Dash doesn't have any built in alerting capabilities but it is possible to construct your own alerts based off the data collected.  I've included some examples which might be useful.

At some point DBA Dash might be extended to have some built in alerting capabilities but there are **no immediate plans** to implement this.  

I would be interested in feedback from the community around alerting.  How should it work and what sort of things people are interested in alerting on.  I have my own ideas, but alerting can be a tricky thing to get right.  Most products don't provide good alerting out the box from my experience and you need to spend time customizing them to your environment.  If you get the setup wrong and generate too many alerts that can be as bad as having no alerts.

Hope this helps,

David

**Comment by gayazics** [comment]
Thank you David.. yes, my question is answered.

Regards,
Gayaz

**Comment by alinselicean** [comment]
Hi all
Although an old discussion (well, not so old :) ), I thought it might help to know that I've added my take on the alerting topic here: https://github.com/trimble-oss/dba-dash/discussions/516

It's my own implementation (with some external inspirations), but nonetheless, as @DavidWiseman mentioned above, it can be tricky to get it right. It's still WIP, so take it as it is.

HTH

Alin

**Comment by DavidWiseman** [comment]
Alerts are now available for beta testing in [3.17](https://dbadash.com/blog/whats-new-in-3.17/)
